Source/ThirdParty/ik/CMakeLists.txt

```diff
@@ -38,19 +34,20 @@ option (IK_MEMORY_DEBUGGING "Global switch for memory options. Keep track of the
 cmake_dependent_option (IK_MEMORY_BACKTRACE "Generate backtraces for every malloc(), making it easy to track down memory leaks" "${DEFAULT_MEMORY_DEBUGGING}" "IK_MEMORY_DEBUGGING AND NOT WEB" FALSE)
 
 # Enable restrict keyword in quaternion and vector operations if not in debug
-set (IK_RESTRICT "")
-if (NOT CMAKE_BUILD_TYPE STREQUAL Debug)
+if (NOT IK_RESTRICT)
+    set (IK_RESTRICT "")
```

Comment:
Is there a way to check if a cmake variable is cached or not? Ideally that's what should be there
```cmake
if (IK_RESTRICT ISNT CACHED)
    ... do the stuff ...
endif ()
```